src/verifier.ts

```diff
@@ -4,11 +4,34 @@ import CERTIFICATE_VERSIONS, { isV3 } from './constants/certificateVersions';
 import VerifierError from './models/verifierError';
 import domain from './domain';
 import * as inspectors from './inspectors';
+import { Blockcerts } from './models/Blockcerts';
+import { ExplorerAPI } from './certificate';
 
 const log = debug('Verifier');
 
+export interface IVerificationStepCallbackAPI {
+  code: string;
+  label: string;
+  status: string; // enum?
+  errorMessage?: string;
+}
+
+export type IVerificationStepCallbackFn = (update: IVerificationStepCallbackAPI) => any;
```

Comment:
this type needs to be exposed to the consumer.

Reply:
👍